openfe/protocols/openmm_utils/omm_settings.py

```diff
@@ -345,17 +336,51 @@ class Config:
     """
     Frequency to write the checkpoint file. Default 250 * unit.timestep.
     """
-    checkpoint_storage = 'checkpoint.nc'
+    checkpoint_storage_filename = 'checkpoint.chk'
     """
     Separate filename for the checkpoint file. Note, this should
-    not be a full path, just a filename. Default 'checkpoint.nc'.
+    not be a full path, just a filename. Default 'checkpoint.chk'.
     """
     forcefield_cache: Optional[str] = 'db.json'
     """
     Filename for caching small molecule residue templates so they can be
     later reused.
     """
 
+    @validator('checkpoint_interval')
+    def must_be_positive(cls, v):
+        if v <= 0:
+            errmsg = "Checkpoint intervals must be positive"
+            raise ValueError(errmsg)
+        return v
+
+class SimulationSettings(SettingsBaseModel):
+    """
+    Settings for simulation control, including lengths,
+    writing to disk, etc...
+    """
+    class Config:
+        arbitrary_types_allowed = True
+
+    minimization_steps = 5000
```

Comment:
unit.timestep?

Reply:
Minimization is not a timestep - it's a unitless quantity because it's a pure count of the cycles of your minimizer.